### ISSUE TITLE
Github page for landing

### ISSUE THREAD

**Issue by lqez** [keycastr/keycastr#15]
It would be pretty good for new users - yes, os x end-users - if we have a landing page with below items.
- Description for keycastr
- Document link (to wiki)
- Download link (to release)
- Issue link (to here)
- License information
- Contributors and history of keycastr

**Comment by phanimahesh** [comment]
Yep that. I just landed on the github project because someone I know starred it recently. And I have no clue what this does, and can't dive deep right now. A short description would be great!

**Comment by sdeken** [comment]
We definitely need a more robust set of documentation.  We don't have a very good elevator pitch right now, and we don't have documentation for a lot of the things that people are tripping up on, like how to move the visualizers around.  Setting a 1.0 milestone - it's not done until this is done!

**Comment by davidhcefx** [comment]
IMO the current landing page is good enough! https://github.com/keycastr/keycastr/blob/58133a7bca58586b6fa1f462b4c13f09404f9c6a/README.md
